packages/chrome/styleguide.config.js

```diff
@@ -5,42 +5,72 @@
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
+const path = require('path');
+const reactDocgenTypescript = require('react-docgen-typescript');
+const reactDocgen = require('react-docgen');
+
 /**
  * Package specific styleguide configuration
  * https://github.com/styleguidist/react-styleguidist/blob/master/docs/Configuration.md
  */
 module.exports = {
+  propsParser: reactDocgenTypescript.withCustomConfig(
+    path.resolve(__dirname, '../../tsconfig.json'),
+    {
+      propFilter: props => {
+        return props.parent.fileName.indexOf('node_modules') === -1;
+      }
+    }
+  ).parse,
+  resolver: reactDocgen.resolver.findAllComponentDefinitions,
   sections: [
     {
       name: '',
       content: '../../packages/chrome/README.md'
     },
     {
-      name: 'Views',
+      name: 'Examples',
+      sections: [
+        {
+          name: 'Basic',
+          content: '../../packages/chrome/examples/basic.md'
+        },
+        {
+          name: 'Header',
+          content: '../../packages/chrome/examples/header.md'
+        },
+        {
+          name: 'Sidebar',
+          content: '../../packages/chrome/examples/sidebar.md'
+        }
+      ]
+    },
+    {
+      name: 'Elements',
       sections: [
         {
           name: 'Chrome',
-          components: '../../packages/chrome/src/views/[A-Z]*.js'
+          components: '../../packages/chrome/src/elements/[A-Z]*.{ts,tsx}'
```

Comment:
Pull the `Chrome` component out of a section and just place directly under "Elements". I think if you specify a `components` key at the same level as `name: 'Elements'` this might be possible.

Reply:
This ends up putting it at the bottom, same as the `elements` block 😢 

I was able to provide an array of component globs which moves the content and `Chrome` components under `Elements`

```
      name: 'Elements',
      components: [
        '../../packages/chrome/src/elements/[A-Z]*.{ts,tsx}',
        '../../packages/chrome/src/elements/body/[A-Z]*.{ts,tsx}'
      ],
```